infra/gcb/templates/src/build-status/build-status.html

```diff
@@ -26,17 +41,36 @@
       data="{{routeData}}">
     </app-route>
     <style is="custom-style" include="iron-flex iron-flex-alignment">
-    <style>
+    <style> 
       .paper-item-link {
         color: inherit;
         text-decoration: none;
       }
 
+      a {
+        text-decoration: none;
+      }
+
       app-header {
         background-color: #2ba4ad;
         color: #fff;
       }
 
+      paper-button {
+        font-weight: normal;
+        font-size: 14px;
+        -webkit-font-smoothing: antialiased;
```

Comment:
is this really needed?

Reply:
Buttons just look a little better with these settings